rsts/getting_started/index.rst

```diff
@@ -1,48 +1,145 @@
+
+    @workflow
+    def wf(n: int = 200, mean: float = 0.0, sigma: float = 1.0) -> typing.Tuple[float, float]:
+        return compute_stats(df=generate_normal_df(n=n, mean=mean, sigma=sigma))
+
+
+Running Flyte Workflows
+^^^^^^^^^^^^^^^^^^^^^^^
+You can run the workflows in ``example.py`` on a local python environment or a Flyte cluster. In this guide, you'll learn how to run a demo local cluster.
+
+Executing Locally
+"""""""""""""""""""
+
+Run your workflow locally using ``pyflyte``, which is the CLI that ships with ``flytekit``.
+
+.. prompt:: bash $
+
+  pyflyte run example.py:wf --n 500 --mean 42 --sigma 2
+
+.. dropdown:: :fa:`info-circle` Why ``pyflyte run`` - You can simply use ``python example.py`` - But?
+    :animate: fade-in-slide-down
+
+    Flyte ``@task`` and ``@workflow`` decorators are designed to continue to work with your code-base. There is one
+    restriction, they have to be the outermost decorators. Thus you can, simply invoke your ``tasks`` and ``workflows``
+    as regular python methods.
+
+    Also note, that Tasks are pure python functions, while Workflows are a DSL and only look like a python function.
+    Do not use non-determininstic operations in a workflow - like ``rand.random`` or ``time.now()`` etc. Also, in a workflow
+    you cannot simply access the outputs of tasks and operate on them, you can only pass them to other tasks.
+
+Executing on a Flyte Cluster
+"""""""""""""""""""""""""""""""
+
+Install :std:ref:`flytectl`, which is the command-line interface for Flyte. Let's use it to start a local demo Flyte cluster.
+
+.. tabbed:: OSX
+
+  .. prompt:: bash $
+
+    brew install flyteorg/homebrew-tap/flytectl
+
+.. tabbed:: Other Operating systems
+
+  .. prompt:: bash $
+
+    curl -sL https://ctl.flyte.org/install | sudo bash -s -- -b /usr/local/bin # You can change path from /usr/local/bin to any file system path
+    export PATH=$(pwd)/bin:$PATH # Only required if user used different path then /usr/local/bin
+
+
+Start a Flyte demonstration environment on your local machine:
+
+.. prompt:: bash $
+
+  flytectl demo start
+
+
+.. dropdown:: :fa:`info-circle` What is a flyte demo environment?
+    :animate: fade-in-slide-down
+
+    Flyte demo environment is a fully included testing environment that can be run on your local machine. It is not a substitute for production environment
+    but is great to try out the platform and check out some of the capabilities. Most plugins are not directly installed in this environment and it is not really
+    a great way to test the performance of the platform.
+
+Then run the same workflow on the Flyte cluster:
+
+.. prompt:: bash $
+
+  pyflyte run --remote example.py:wf --n 500 --mean 42 --sigma 2
+
+.. dropdown:: :fa:`info-circle` What does the ``--remote`` flag do?
+    :animate: fade-in-slide-down
+
+   * The only difference between previous ``local`` and this command is the ``--remote`` flag. This will trigger an execution on the configured backend.
```

Comment:
```suggestion
   * The only difference between the previous ``local`` and this command is the ``--remote`` flag. This will trigger the execution on the configured backend.
```